tests/sentry/replays/test_organization_replay_count.py

```diff
@@ -413,36 +416,6 @@ def test_simple_transaction(self):
         assert response.status_code == 200, response.content
         assert response.data == expected
 
-    def test_max_51(self):
```

Comment:
Is this coverage no longer applicable?

Reply:
woops, meant to comment here. its a really slow test, and i don't think gets us much.